testdata/go/v3/monitoring/memcached-operator/monitoring/metricsdocs/metricsdocs.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/example/memcached-operator/monitoring"
+)
+
+// constant parts of the file
```

Comment:
Can we add here, that it is the operator author's responsibility to update this doc, so as to enable uses of their project receive updated documentation on metrics.

Reply:
you mean like in metrics.md (mentioned in the comment above)?

Reply:
Done.

Added:

```suggestion

// please run \"make generate-metricsdocs\" to generate this document.
// constant parts of the file
```